src/Core/Domain/Meta/DataTransferObject/LayoutCustomizationPage.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * 2007-2018 PrestaShop.
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Core\Domain\Meta\DataTransferObject;
```

Comment:
Is it the 1st time you use DataTransferObjects ? Do you want to start using theme more ?

Reply:
what do you mean? :confused:

Reply:
DataTransferObject is a concept that can be enforced in a php project, the same way Form Factories or Data Handlers are now enforced for SF controllers in migrated PrestaShop.

So I wanted to know if you wanted to introduce the concept in PrestaShop using this PR, and start using them everywhere.

Reply:
i dont really think its a "new" concept in PS, there are `Dto`s already, except they are not in explicit namespace. :)